twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts

```diff
@@ -33,30 +37,66 @@ export const transformValueToDbString = (v: any, type: ColumnType, options: any
     }
     return `${!!v}`;
   }
-  if (type === "string" || type === "json") {
+  if ( v !== null && type === "string" || type === "json" ) {
     if (type === "json") {
       try {
         v = JSON.stringify(v);
       } catch (err) {
         v = null;
       }
     }
-    return `'${v || ""}'`; //Encryption not implemented yet
+    
+    let iv = randomBytes(16);
+    var encrypt = ((val: any) => {
+      let cipher = crypto.createCipheriv('aes-256-cbc', encryptionKey, iv);
+      return Buffer.concat([
+        cipher.update(val),
+        cipher.final()
+      ]);
+    });
+    console.log("A ENCODER: ", v);
+    console.log("TRUC STOCKEE EN BD: ",  iv.toString('hex') + ":" + encrypt(v).toString('hex'));
```

Comment:
Remove such comments

Reply:
Forgot to remove all my console.log, I will remove them.